tests/objc-cli/test-managed.m

```diff
@@ -22,17 +22,21 @@ int main ()
 		NSLog (@"%s instance property getter", [query answer] == 42 ? "[PASS]" : "[FAIL]");
 		query.answer = 911;
 		NSLog (@"%s instance property setter", [query answer] == 911 ? "[PASS]" : "[FAIL]");
+		[query release];
```

Comment:
it may be harsh but I rather support less now (and _possibly_ more later) and focus on the called (not the caller) code :)